sqlitecluster/SQLite.cpp

```diff
@@ -1067,6 +1071,15 @@ bool SQLite::getUpdateNoopMode() const {
     return _noopUpdateMode;
 }
 
+int SQLite::timedSQuery(const char *e, const string &sql, SQResult &result, int64_t warnThreshold, bool skipWarn) {
+    uint64_t startTime = STimeNow();
+    int queryResult = SQuery(_db, e, sql, result, skipWarn);
+    uint64_t elapsed = STimeNow() - startTime;
```

Comment:
I'm going to suggest using the new std::chrono functions like we use here: https://github.com/Expensify/Bedrock/blob/master/main.cpp#L315-L350

But if you like it this way feel free to ignore me.